### PR TITLE
[Improve] Provide Maturity Assessment for Apache StreamPark

### PR DESCRIPTION
**Pull request by wolfboys**

### CODE REVIEW

File: community/maturity.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+| **QU20** | The project puts a very high priority on producing secure software.                                                                                                                           | **YES** We will promptly confirm any security issues. Currently, All reported security issues have been resolved.                                                                      |
+| **QU30** | The project provides a well-documented, secure and private channel to report security issues, along with a documented way of responding to them.                                              | **Yes** Yes The official website provides a [Security page](https://streampark.apache.org/community/maturity）                                                                                                                |
```
Comment (tisonkun) [inline community/maturity.md]:
```suggestion
| **QU30** | The project provides a well-documented, secure and private channel to report security issues, along with a documented way of responding to them.                                              | **Yes** Yes The official website provides a [Security page](https://streampark.apache.org/community/security）                                                                                                                |
```

File: community/maturity.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+| **CO10** | The project has a well-known homepage that points to all the information required to operate according to this maturity model.                                         | **YES** The [official website](https://streampark.apache.org/) includes all information user need to run Apache StreamPark.                                                                                                                                                                                        |
+| **CO20** | The community welcomes contributions from anyone who acts in good faith and in a respectful manner, and who adds value to the project.                                 | **Yes** We provide [Contribution Guide](https://streampark.apache.org/community/contribution_guide/mailing_lists/) and [code style](https://streampark.apache.org/community/submit_guide/code_style_and_quality_guide). And we also have a [SPIP](https://streampark.apache.org/community/contribution_guide/SPIP) |
```
Comment (tisonkun) [inline community/maturity.md]:
```suggestion
| **CO20** | The community welcomes contributions from anyone who acts in good faith and in a respectful manner, and who adds value to the project.                                 | **Yes** We provide [Contribution Guide](https://streampark.apache.org/community/contribution_guide/mailing_lists/) and [code style](https://streampark.apache.org/community/submit_guide/code_style_and_quality_guide). And we also have [StreamPark Improvement Proposals (SPIPs)](https://streampark.apache.org/community/contribution_guide/SPIP) |
```

File: community/maturity.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+| **CO40** | The community strives to be meritocratic and gives more rights and responsibilities to contributors who, over time, add value to the project.                          | **YES** The community has elected 2 new PPMC members and 8 new committers so far.                                                                                                                                                                                                                                  |
+| **CO50** | The project documents how contributors can earn more rights such as commit access or decision power, and applies these principles consistently.                        | **YES** The community has clear docs on [nominatin committers](https://streampark.apache.org/community/contribution_guide/become_committer) and PPMC members                                                                                                                                                       |
```
Comment (tisonkun) [inline community/maturity.md]:
```suggestion
| **CO50** | The project documents how contributors can earn more rights such as commit access or decision power, and applies these principles consistently.                        | **YES** The community has clear docs on [nominatin committers](https://streampark.apache.org/community/contribution_guide/become_committer) and PPMC members.                                                                                                                                                       |
```

**Review by tisonkun — APPROVED** [review]
Rest LGTM. We can land it and start the general discussion for graduation then.

**Comment by wolfboys** [comment]
> Rest LGTM. We can land it and start the general discussion for graduation then.

Thanks for your review. I will be launching a graduation discussion soon

**Pull request merged**